src/overlays/actors/ovl_En_Test7/z_en_test7.c

```diff
@@ -988,6 +988,6 @@ void EnTest7_Draw(Actor* thisx, PlayState* play) {
 
     if (this->unk_144 & 4) {
         Environment_DrawLensFlare(play, &play->envCtx, &play->view, play->state.gfxCtx, this->actor.world.pos, 70.0f,
```

Comment:
EnInvadepohDemo also has a call to this function that needs the last arg changed.